From fae86f07a5971338af6788d98009e12324f8a509 Mon Sep 17 00:00:00 2001
From: Tzafrir Cohen <tzafrir@debian.org>
Date: Thu, 30 Oct 2014 08:09:44 +0200
Subject: Enable building python bindings

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 20af077..1dfac1f 100644
--- a/Makefile
+++ b/Makefile
@@ -4,6 +4,7 @@ include build/host-$(HOST_NAME).mak
 include version.mak
 
 LIB_DIRS = pjlib/build pjlib-util/build pjnath/build third_party/build pjmedia/build pjsip/build
+EXTRA_DIRS = pjsip-apps/src/python
 DIRS = $(LIB_DIRS) pjsip-apps/build $(EXTRA_DIRS)
 
 ifdef MINSIZE
